Cargo.toml

```diff
@@ -50,7 +50,13 @@ check_asm = []
 capi = []
 tracing = ["rust_hawktracer/profiling_enabled"]
 scenechange = []
-serialize = ["serde", "toml", "v_frame/serialize", "arrayvec/serde"]
+serialize = [
+    "serde",
+    "toml",
+    "v_frame/serialize",
+    "arrayvec/serde",
```

Comment:
Is this crate still in use?

Reply:
Oh good catch, yeah it's not in use anymore.